tests/suites/test_suite_aes.function

```diff
@@ -369,6 +370,54 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE depends_on:MBEDTLS_VALIDATE_AES_KEYS:MBEDTLS_AES_SCA_COUNTERMEASURES:!MBEDTLS_AES_SETKEY_ENC_ALT:!MBEDTLS_AESNI_C */
+void aes_encrypt_ecb_crc( data_t * key_str, data_t * src_str,
+                          data_t * hex_dst_string, unsigned int crc, int crypt_result )
+{
+    unsigned char output[100];
+    mbedtls_aes_context ctx;
+
+    memset(output, 0x00, 100);
+
+    mbedtls_aes_init( &ctx );
+
+    TEST_ASSERT( mbedtls_aes_setkey_enc( &ctx, key_str->x, key_str->len * 8 ) == 0 );
+
+    ctx.crc = crc;
```

Comment:
@AndrzejKurek , why do you set crc to ctx? Did you mean ctx.crc == crc?

Reply:
No, I wanted to perform a test where the hypothetical external user can change the crc to simulate fault injection. There's test data that has cases where this CRC is wrong, and the test case ensures that a FAULT is received.

Reply:
I think this is a good test for possible attacks by fault injection. On the other this test does not check if `ctx.crc` is calculated properly by `mbedtls_aes_setkey_enc()`. It might be a good idea to add an additional ASSERT with checking
```suggestion
    TEST_ASSERT( ctx.crc = crc_to_check );
    ctx.crc = crc_to_set;
```

Reply:
Good suggestion, @piotr-now. Added.